test/e2e-smoke-tests.sh

```diff
@@ -46,16 +46,14 @@ header "Running smoke tests"
 
 kubectl create ns $KN_E2E_SMOKE_TESTS_NAMESPACE || fail_test
 
+./kn service create svc1 --async --image gcr.io/knative-samples/helloworld-go -e TARGET=Knative -n $KN_E2E_SMOKE_TESTS_NAMESPACE || fail_test
```

Comment:
Since you also added an `--async` test just modify commit message to mention that and should lgtm

Reply:
[Commit](https://github.com/knative/client/pull/220/commits/d9e7f2975c698ebbc1270e7b0924dbf53731a5f6) message has it
```
Removes pausing between e2e smoke tests

 Removes time.Sleep as by default service create waits for it to
 become ready. Also adds a command to create service in --async mode.
```